plugin/jumpmotion.vim

```diff
@@ -8,7 +8,7 @@ endif
 let s:save_cpo = &cpo
 set cpo&vim
 
-if !hasmapto('<Plug>(JumpMotion)')
+if get('g:jumpmotion_do_mappings', 1) == 1 && !hasmapto('<Plug>(JumpMotion)')
```

Comment:
I think it would be `get(g:, 'jumpmotion_do_mappings', 1)` correctly. `g:` is the list of all globals, and '...' is the key we are looking for.

Reply:
Fixed, sorry for the long delay on such a simple change!